Cargo.toml

```diff
@@ -22,9 +22,9 @@ edition.workspace = true
 repository.workspace = true
 
 [workspace.dependencies]
-vello = { git = "https://github.com/linebender/vello", rev = "b0303ccf98df15a8b196272720d364a56f7304ba" }
+vello = { git = "https://github.com/linebender/vello.git", rev = "34d313d0decc6c3ddbb9185195356ce5accde305", default-features = false }
 
 
 [dependencies]
 vello = { workspace = true }
-bodymovin = { git = "https://github.com/vectorgameexperts/bodymovin-rs" }
+bodymovin = { git = "https://github.com/Traverse-Research/bodymovin-rs.git" }
```

Comment:
This needs to use a specific `rev`, as is linebender policy (I don't think this is written down anywhere). This should have been set previously as well. You can understand that this is especially important for external code.

I also think the PR body should explain the differences in this fork

Reply:
Both the `vectorgameexperts` and `Traverse-Research` forks seem to be identical with HEAD at `e65aea9a701a0c934382ae0037343ecb0ac6976a`. We should certainly pin to that rev and I guess which fork to use doesn't matter too much when it's pinned.

Reply:
Yeah I made the fork for no real reason, so far, I can revert this also. But we should indeed pin to the right hash.